### PR TITLE
Add 600 series pch config

### PR DESCRIPTION
**Pull request by dscott90**
In response to https://github.com/chipsec/chipsec/issues/1542

### CODE REVIEW

File: chipsec/cfg/8086/pch_6xxS.xml [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+      <field name="IOS"   bit="0" size="1"  desc="I/O space"/>
+      <field name="TCOBA" bit="5" size="11" desc="TCO Base Address"/>
+    </register>
+    <register name="TCOCTL"     type="pcicfg" bus="0" dev="0x1f" fun="4" offset="0x54" size="4" desc="TCO Control">
+      <field name="TCO_BASE_LOCK" bit="0" size="1" desc="TCO Base Lock"/>
+      <field name="TCO_BASE_EN"   bit="8" size="1" desc="TCO Base Enable"/>
+    </register>
+
+    <!-- SPI Interface Controller -->
+    <register name="BC" type="pcicfg" bus="0" dev="0x1F" fun="5" offset="0xDC" size="4" desc="BIOS Control">
```
Comment (npmitche) [inline chipsec/cfg/8086/pch_6xxS.xml:225]:
```suggestion
    <register name="BC" type="pcicfg" device="SPI" offset="0xDC" size="4" desc="BIOS Control">

```

File: chipsec/cfg/8086/pch_6xxS.xml [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+      <field name="BA"    bit="7" size="25" desc="Base Address"/>
+    </register>
+
+    <!-- Intel HD Audio -->
+    <register name="HDABAR" type="pcicfg" bus="0" dev="0x1f" fun="3" offset="0x10" size="4" desc="HD Audio Base Lower Address" >
+        <field name="BA" bit="14" size="18" desc="Lower Base Address"/>
+    </register>
+
+    <!-- SMBus Host Controller -->
+    <register name="SMB_BASE" type="pcicfg" bus="0" dev="0x1f" fun="4" offset="0x20" size="4" desc="" >
```
Comment (npmitche) [inline chipsec/cfg/8086/pch_6xxS.xml:202]:
```suggestion
    <register name="SMB_BASE" type="pcicfg" device="SMBUS" offset="0x20" size="4" desc="" >

```
Include this change for the below registers too.

File: chipsec/cfg/8086/pch_6xxP.xml [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+      <field name="IOS"   bit="0" size="1"  desc="I/O space"/>
+      <field name="TCOBA" bit="5" size="11" desc="TCO Base Address"/>
+    </register>
+    <register name="TCOCTL"     type="pcicfg" bus="0" dev="0x1f" fun="4" offset="0x54" size="4" desc="TCO Control">
+      <field name="TCO_BASE_LOCK" bit="0" size="1" desc="TCO Base Lock"/>
+      <field name="TCO_BASE_EN"   bit="8" size="1" desc="TCO Base Enable"/>
+    </register>
+
+    <!-- SPI Interface Controller -->
+    <register name="BC" type="pcicfg" bus="0" dev="0x1F" fun="5" offset="0xDC" size="4" desc="BIOS Control">
```
Comment (npmitche) [inline chipsec/cfg/8086/pch_6xxP.xml:203]:
```suggestion
    <register name="BC" type="pcicfg" device="SPI" offset="0xDC" size="4" desc="BIOS Control">

```

File: chipsec/cfg/8086/pch_6xxP.xml [hunk 4]
```diff
@@ -0,0 +1,411 @@
...
+      <field name="BA"    bit="7" size="25" desc="Base Address"/>
+    </register>
+
+    <!-- Intel HD Audio -->
+    <register name="HDABAR" type="pcicfg" bus="0" dev="0x1f" fun="3" offset="0x10" size="4" desc="HD Audio Base Lower Address" >
+        <field name="BA" bit="14" size="18" desc="Lower Base Address"/>
+    </register>
+
+    <!-- SMBus Host Controller -->
+    <register name="SMB_BASE" type="pcicfg" bus="0" dev="0x1f" fun="4" offset="0x20" size="4" desc="" >
```
Comment (npmitche) [inline chipsec/cfg/8086/pch_6xxP.xml:180]:
```suggestion
    <register name="SMB_BASE" type="pcicfg" device="SMBUS" offset="0x20" size="4" desc="" >

```
Make the change also in the registers below.

File: chipsec/cfg/8086/pch_6xxP.xml [hunk 5]
```diff
@@ -0,0 +1,411 @@
...
+    <!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+    <!-- MSR Definitions              -->
+    <!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+
+    <!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+    <!-- PCIe Configuration registers -->
+    <!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+
+    <!-- Sideband Register Access Registers -->
+    <register name="SBREG_BAR" type="pcicfg" bus="0" dev="0x1F" fun="1" offset="0x10" size="4" desc="Sideband Register Access BAR">
```
Comment (npmitche) [inline chipsec/cfg/8086/pch_6xxP.xml:93]:
```suggestion
    <register name="SBREG_BAR" type="pcicfg" device="P2SBC" offset="0x10" size="4" desc="Sideband Register Access BAR">

```
Same with below register

**Review by frinzell — APPROVED** [review]

**Review by npmitche — APPROVED** [review]

**Pull request merged**